doc/DISCLAIMER.md

```diff
@@ -1,6 +0,0 @@
-* Only installable on a domain root (or a subdomain root)!
```

Comment:
-> https://github.com/YunoHost/apps/pull/1685

Reply:
> It would be nice to have in https://github.com/YunoHost/apps/blob/master/antifeatures.toml a "requires its own domain" key :')

There is a setting in manifest I think

Reply:
I think you mean "some webapp do require a full dedicated domain and do not support the "subpath" install scheme. In that case, you typically want to remove the path question entirely" ?

I was talking about a real disclaimer on the "store" page of the app.